src/Microsoft.ML.Featurizers/DateTimeTransformer.cs

```diff
@@ -346,46 +341,17 @@ public TransformedDataSafeHandle(IntPtr handle, DestroyTransformedDataNative des
             protected override bool ReleaseHandle()
             {
                 // Not sure what to do with error stuff here.  There shouldn't ever be one though.
-                return _destroyTransformedDataHandler(handle, out IntPtr errorHandle);
+                var success = _destroyTransformedDataHandler(handle, out IntPtr errorHandle);
+                Marshal.FreeHGlobal(handle); // Free the memory allocated in C#.
+                return success;
             }
         }
 
         #endregion
 
-        #region TimePoint
-
-        // Exact native representation
-        [StructLayout(LayoutKind.Sequential, Pack = 1)]
-        internal struct NativeTimePoint
```

Comment:
Why is packing no longer necessary? Has the layout changed on the native side?

Reply:
Per offline discussion this whole section needs to be re-added in. It was an accidental removal due to a rebase error.